low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml

```diff
@@ -30,6 +30,17 @@ cluster:
          ip: 127.0.0.1
          secret: gAAAAABgEnOnRAhNzFyATr3iZdoC_tCeqEBqnwFb7CNYVXhd6i3Pl93Il7aDsG0Ro5ZRRaXr2LUPQEse46NVlDTAK7gbzoxGyg==
          user: admin
+      roles:
```

Comment:
what this role signify? And what these openldap_server, kafka_server mean here really

Reply:
kafka_server is used by `utils_setup config`(if node is part of messaging cluster or not). Removed openldap_server, primary as it not needed by SSPL or utils

Reply:
roles?